crates/ra_hir_expand/src/name.rs

```diff
@@ -62,6 +62,13 @@ impl Name {
             _ => None,
         }
     }
+
+    pub fn as_text(&self) -> Option<&SmolStr> {
```

Comment:
We intentionally don't expose `&str` out of `Name`, as that can't work properly with hygiene (once we add it). The correct way to get string data out of `Name` is the `fmt::Dispaly` impl.

Reply:
Thanks for the tip. I reverted this.